### DISCUSSION TITLE
LTBEEF inspect "Patch" [Securly]

### DISCUSSION THREAD

**Discussion by alcradite** [3kh0/ext-remover#650] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So this is with Securly and well even when fully disabling the extension it is somewhat still able to process certain links with the certificate manager [Our Certificate Manager is not able to be edited] and so when you go to a link that is usually blocked it comes up with the following image 
![image](https://github.com/3kh0/ext-remover/assets/133121631/160a2c2f-9073-4a02-b81f-47623dc21ec1)
and when you process it brings you to
"https://useast2-www.securly.com/blocked.php?gnp=1&reason=domainblockedforuser&url=_thelinkyoutrytoaccess_"
But some links that usually would be blocked like "Discord", "Soundcloud", "Spotify" is not blocked

I was wondering if there was a fix for this

**Comment by S-PScripts** [comment, marked as answer]
test
